sdk/communication/azure-communication-sms/README.md

```diff
@@ -54,23 +54,44 @@ endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_ENDPOINT')
 sms_client = SmsClient(endpoint, DefaultAzureCredential())
 ```
 
-### Sending SMS Messages
+### Send a 1:1 SMS Message
 
 Once the client is initialized, the `.send()` method can be invoked:
 
 ```Python
 from azure.communication.sms import SendSmsOptions
 
 sms_responses = sms_client.send(
-    from_phone_number=PhoneNumberIdentifier("<leased-phone-number>"),
-    to_phone_numbers=["<to-phone-number-1>", "<to-phone-number-2>", "<to-phone-number-3>"],
+    from_="<leased-phone-number>",
```

Comment:
Just noticed this, is it intentional that "from_" ends with an underscore? Probably an ApiView question if anything

Reply:
Yeah, that's intentional since ```from``` is a special keyword in python, so we cant use it. We discussed this with the arch board folks and decide to go with the underscore suffix.